tests/usb/usb.go

```diff
@@ -103,6 +106,22 @@ var _ = Describe("[Serial][sig-compute][USB] host USB Passthrough", Serial, deco
 				&expect.BSnd{S: fmt.Sprintf("%s\n", cmdNumberUSBs)},
 				&expect.BExp{R: console.RetValue(fmt.Sprintf("%d", len(deviceNames)))},
 			}, 15)).To(Succeed(), "Device not found")
+
+			By("Verifying ownership is properly set in the host")
+			domainXml, err := tests.GetRunningVMIDomainSpec(vmi)
+			Expect(err).ToNot(HaveOccurred())
+
+			for _, hostDevice := range domainXml.Devices.HostDevices {
+				if hostDevice.Type != api.HostDeviceUSB {
+					continue
+				}
+				addr := hostDevice.Source.Address
+				path := fmt.Sprintf("%sdev/bus/usb/00%s/00%s", pkgUtil.HostRootMount, addr.Bus, addr.Device)
+				cmd := []string{"stat", "-c", `"%u %g"`, path}
+				stdout, err := tests.ExecuteCommandInVirtHandlerPod(nodeName, cmd)
```

Comment:
Shouldn't this `nodeName` come from VMI `status`?

Reply:
```diff
-                               stdout, err := tests.ExecuteCommandInVirtHandlerPod(nodeName, cmd)
+                               stdout, err := tests.ExecuteCommandInVirtHandlerPod(vmi.Status.NodeName, cmd)
```

aha. could this be the reason for [previous failure](https://github.com/kubevirt/kubevirt/pull/10753#issuecomment-1820214901)?

Thanks for catching this.

Reply:
`vmi.Status.NodeName` was "", I missed updating the vmi variable after start, so the change actually is
```diff
--- a/tests/usb/usb.go
+++ b/tests/usb/usb.go
@@ -98,7 +98,7 @@ var _ = Describe("[Serial][sig-compute][USB] host USB Passthrough", Serial, deco
                        vmi.Spec.Domain.Devices.HostDevices = hostDevs
                        vmi, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(context.Background(), vmi)
                        Expect(err).ToNot(HaveOccurred())
-                       libwait.WaitForSuccessfulVMIStart(vmi)
+                       vmi = libwait.WaitForSuccessfulVMIStart(vmi)
                        Expect(console.LoginToCirros(vmi)).To(Succeed())

                        By("Making sure the usb is present inside the VMI")
@@ -118,9 +118,9 @@ var _ = Describe("[Serial][sig-compute][USB] host USB Passthrough", Serial, deco
                                addr := hostDevice.Source.Address
                                path := fmt.Sprintf("%sdev/bus/usb/00%s/00%s", pkgUtil.HostRootMount, addr.Bus, addr.Device)
                                cmd := []string{"stat", "-c", `"%u %g"`, path}
-                               stdout, err := tests.ExecuteCommandInVirtHandlerPod(nodeName, cmd)
+                               stdout, err := tests.ExecuteCommandInVirtHandlerPod(vmi.Status.NodeName, cmd)
                                Expect(err).ToNot(HaveOccurred())
```